### PR TITLE
Add package to share Prettier configuration

### PR DESCRIPTION
**Pull request by joaopalmeiro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,50 @@
...
 ```bash
-npx prettier --write .
+yarn add --dev prettier @feedzai/prettier-config
 ```
 
 or
 
 ```bash
-yarn prettier --write .
+pnpm add --save-dev prettier @feedzai/prettier-config
```
Comment (lfac-pt) [inline README.md:26]: `pnpm` should be `npm`

File: README.md [hunk 2]
```diff
@@ -1,58 +1,50 @@
...
+After changes, sort the `package.json` file and run Prettier:
 
-If you use ESLint, install [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier#installation) to make ESLint and Prettier play nice with each other. It turns off all ESLint rules that are unnecessary or might conflict with Prettier. There’s a similar config for Stylelint: [stylelint-config-prettier](https://github.com/prettier/stylelint-config-prettier)
-
-## References
-[Prettier](https://prettier.io)
-[Editor Integration](https://prettier.io/docs/en/editors.html)
-[How to set options in Prettier](https://prettier.io/docs/en/options.html)
+```bash
+npx sort-package-json package.json
```
Comment (lfac-pt) [inline README.md:45]: Why is this needed?

File: README.md (hunk 1 above)
Comment (joaopalmeiro) [reply README.md:26]: I've added commands for `npm` (above), `yarn`, and `pnpm`, so it's easy to copy and paste the command to your main ones. What do you think?

File: README.md (hunk 2 above)
Comment (joaopalmeiro) [reply README.md:45]: Just to format the `package.json` file so that the fields follow a certain order for consistency.

File: README.md (hunk 1 above)
Comment (waldyrious) [reply README.md:26]: That's in line 14. [Pnpm](https://pnpm.io/) is an actual separate package manager, and it has a pretty neat concept! It uses content-addressable storage (like git) to deduplicate all the node_modules contents globally. Check it out!

Comment (lfac-pt) [reply README.md:26]: Ah ok. Sorry, I thought it was a typo. My bad :)
